src/sentry/static/sentry/app/components/stream/groupChart.tsx

```diff
@@ -2,33 +2,34 @@ import LazyLoad from 'react-lazyload';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import {Group, GroupStats} from 'app/types';
 import BarChart from 'app/components/barChart';
 
-class GroupChart extends React.Component {
+const defaultProps = {
+  height: 24,
+};
+
+type Props = {
+  statsPeriod: string;
+  data: Group;
+  hasDynamicIssueCounts?: boolean;
+} & typeof defaultProps;
+
+class GroupChart extends React.Component<Props> {
```

Comment:
as you are no longer using any life cycle method + the class doesn't use state, I'd convert it to functional component... but it's only a suggestion...